plugin/trino-prometheus/src/test/java/io/trino/plugin/prometheus/TestPrometheusColumnHandle.java

```diff
@@ -32,19 +30,4 @@ public void testJsonRoundTrip()
         PrometheusColumnHandle copy = COLUMN_CODEC.fromJson(json);
         assertThat(copy).isEqualTo(columnHandle);
     }
-
-    @Test
-    public void testEquivalence()
```

Comment:
Removed this test because I can't find the reason to use only column name field in `PrometheusColumnHandle.equals`.

Reply:
lgtm